pom.xml

```diff
@@ -1098,12 +1068,26 @@
                   <artifactId>maven-antrun-plugin</artifactId>
                   <version>3.1.0</version>
                   <executions>
+                      <execution>
+                          <id>get-buildvers</id>
+                          <goals>
+                              <goal>run</goal>
+                          </goals>
```

Comment:
There's no `phase` specified for this execution.  We should be explicit on the phase.

Reply:
Because this isn't called during any phase. The only time this target is called is from version-def and buildall scripts